Makefile

```diff
@@ -33,14 +33,58 @@ help:
 
 .PHONY: help
 
+# these are various tools required by make commands below
+GH := $(shell command -v gh 2> /dev/null)
+PYTHON := $(shell command -v python 2> /dev/null)
+JQ := $(shell command -v jq 2> /dev/null)
+
+
 ##############################################################################
-# Release Management
+# Dev Tools Requirements
 ##############################################################################
-VERSION := $(shell node -pe "require('./frontend/package.json').version")
+.PHONY: requirements
+requirements: ## Checks development environment requirements (tools in the PATH)
+ifndef GH
+	$(error "gh (GitHub CLI) is not available please install from https://cli.github.com/")
+endif
+ifndef PYTHON
+	$(error "python3 is not available please install from https://docs.python.org/3")
+endif
+ifndef JQ
+	$(error "jq is not available please install from https://stedolan.github.io/jq")
+endif
+	$(info gh found...)
+	$(info python3 found...)
+	$(info jq found...)
+	@exit 0
+
+.PHONY: check-github-auth
+check-github-auth: ## Checks GITHUB_TOKEN has been set
```

Comment:
GitHub cli needs either GITHUB_TOKEN set in the environment or devs can login via command `gh auth login`